workflow_scripts/check_model.py

```diff
@@ -43,10 +44,13 @@ def run_backend_ort(model_path, test_data_set=None, tar_gz_path=None):
         # For example, if NVIDIA GPU is available and ORT Python package is built with CUDA, then call API as following:
         # onnxruntime.InferenceSession(path/to/model, providers=['CUDAExecutionProvider'])
         onnxruntime.InferenceSession(model_path)
-        ort_test_dir_utils.create_test_dir(model_path, './', test_utils.TEST_ORT_DIR)
-        ort_test_dir_utils.run_test_dir(test_utils.TEST_ORT_DIR)
+        # Get model name without .onnx
```

Comment:
Use https://docs.python.org/3/library/os.path.html#os.path.splitext and https://docs.python.org/3/library/os.path.html#os.path.basename